.github/workflows/Image-build-Egov-Filestore.yaml

```
@@ -0,0 +1,89 @@
+          docker build  -t pucarsolutionsdev.azurecr.io/egov-filestore-db:v1.0.0-${{ env.commit_hash }} -f  Dockerfile .
+          docker push pucarsolutionsdev.azurecr.io/egov-filestore-db:v1.0.0-${{ env.commit_hash }}
+        working-directory: common/egov-filestore/src/main/resources/db
+
+  Workflow_Trigger:
+    if: (github.event_name == 'push' && (github.ref == 'refs/heads/develop' || github.ref == 'refs/heads/uat-backend' || github.ref == 'refs/heads/dpg-qa-deployment')) || github.event_name == 'workflow_dispatch'
+    runs-on: ubuntu-latest
+    needs: docker_image-build
+    steps:
+      - name: Set Workflow File Based on Branch
+        id: set_workflow
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:2:46: Double quote to prevent globbing and word splitting

(shellcheck)